app/src/main/java/com/ataulm/whatsnext/account/SignInActivity.kt

```diff
@@ -29,12 +30,11 @@ class SignInActivity : BaseActivity() {
             screen.setCredentials(BuildConfig.LETTERBOXD_USERNAME, BuildConfig.LETTERBOXD_PASSWORD)
         }
 
-        presenter = SignInPresenter(whatsNextRepository, screen, callback)
+        presenter = SignInPresenter(signInUseCase, screen, callback)
```

Comment:
This SignInActivity isn't accessible to the user anymore. We could delete it I think 🤔 

In the case where both the access and refresh token are not valid (we don't handle this), we could just clear them in the token store, and boot the user back to the SearchActivity (showing the sign in prompt).